api.planx.uk/modules/webhooks/service/sanitiseApplicationData/operations.ts

```diff
@@ -68,8 +68,12 @@ export const getExpiredSessionIds = async (): Promise<string[]> => {
     query GetExpiredSessionIds($retentionPeriod: timestamptz) {
       lowcal_sessions(
         where: {
-          submitted_at: { _lt: $retentionPeriod }
           sanitised_at: { _is_null: true }
+          _or: [
+            { deleted_at: { _lt: $retentionPeriod } }
+            { submitted_at: { _lt: $retentionPeriod } }
+            { locked_at: { _lt: $retentionPeriod } }
+          ]
```

Comment:
Previously, I think this is would have failed to find _un-submitted_ sessions older than the retention period? Do these updated conditions seem correct to others as well?

Reply:
Yep - this makes sense - the previous condition is relying on `submitted_at` being set